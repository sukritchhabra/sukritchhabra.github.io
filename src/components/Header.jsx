import React from 'react';

import 'styles/Header.scss';

const Header = () => (
  <div className="header">
    <h1>ReactTemplate</h1>
  </div>
);

export default Header;
