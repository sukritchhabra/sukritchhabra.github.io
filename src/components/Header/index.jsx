import React from 'react';

import './Header.scss';

const Header = () => (
  <nav className="navbar">
    <h1>Sukrit Chhabra</h1>
  </nav>
);

export default Header;
