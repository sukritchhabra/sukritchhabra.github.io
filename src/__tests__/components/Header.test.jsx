import React from 'react';
import { shallow } from 'enzyme';
import Header from 'components/Header';

describe('Header.jsx', () => {
  const wrapper = shallow(<Header />);

  test('renders', () => {
    expect(wrapper.exists()).toBeTruthy();
    const header = wrapper.find('h1');
    expect(header).toHaveLength(1);
    expect(header.text()).toEqual('ReactTemplate');
  });

  test('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
