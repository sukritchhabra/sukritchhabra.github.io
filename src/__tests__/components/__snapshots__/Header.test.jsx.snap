// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.jsx matches snapshot 1`] = `
<div
  className="header"
>
  <h1>
    ReactTemplate
  </h1>
</div>
`;
